ush/python/pygfs/ufswm/ufs.py

```diff
@@ -104,6 +104,68 @@ def set_ufs_config(self) -> Dict[str, Any]:
 
         return cfg
 
+    def mdl_config_defs(self) -> AttrDict:
+        """
+        Description
+        -----------
+
+        This method assigns the default (i.e., non-resolution)
+        dependent attributes for the UFS `model_configure` file.
+
+        Returns
+        -------
+
+        cfg: AttrDict
+
+            A Python dictionary containing the default
+            `model_configure` attributes values.
+
+        """
+
+        cfg = AttrDict()
+
+        CDATE = datetime_to_YMDH(dt=self._config.CDATE)
+        cfg.SYEAR = CDATE[0:4]
+        cfg.SMONTH = CDATE[5:6]
+        cfg.SDAY = CDATE[7:8]
+        cfg.SHOUR = CDATE[9:10]
```

Comment:
I don't think you need `CDATE` at all.
```suggestion
        cfg.SYEAR = self._config.current_cycle.year
        cfg.SMONTH = self._config.current_cycle.month
        cfg.SDAY = self._config.current_cycle.day
        cfg.SHOUR = self._config.current_cycle.hour
```